db/migrate/20200519141015_create_storage_resources.rb

```diff
@@ -0,0 +1,15 @@
+class CreateStorageResources < ActiveRecord::Migration[5.1]
+  def change
+    create_table :storage_resources do |t|
```

Comment:
`StorageResource` is a little generic, if this is a storage pool I think that would be a more intuitive name.  We also have `PhysicalDisks` which could belong to this new model.

Reply:
StorageResource is an abstract object that may represent a storage pool for certain storages but not necessarily. (It could possibly represent backends, paired pools (for replication) , containers etc)

Reply:
:+1: okay.  A little worried that this being so abstract means it isn't actually representing anything specific but I'm okay seeing where this goes  @Fryguy ?

Reply:
Yeah, I think I'm ok with this for now.  When we get further along and populate it with more than pools, we could always refactor it later.